packages/api-generator/src/index.js

```diff
@@ -262,3 +263,29 @@ components['$vuetify'] = map['$vuetify']
 components['internationalization'] = map['internationalization']
 
 writeApiFile({ ...components, ...directives }, 'dist/api.js')
+
+// Create web-types.json to provide autocomplete in JetBrains IDEs
+const webTypes = {
+  // $schema: "../../schema/web-types.schema.json",
+  framework: "vue",
+  name: "vuetify",
+  version: pkg.version,
+  contributions: {
+    html: {
+      "types-syntax": "typescript",
+      tags: [],
+    }
+  }
+}
+
+const webTypesTags = { ...components, ...directives }
+Object.keys(webTypesTags).forEach(function(key) {
+  const name = key
```

Comment:
names are in kebab case (v-app) right now, but `web-types.json` from JetBrains are using Pascal Case (VApp). @KaelWD I guess I will have to convert case too, will do later.
Meanwhile you can review that everything expect this :point_up_2: is working as expected.
I looked at the generated file, looks ok to me. But it's too long so can't be sure.

Reply:
If there isn't a util for this already you can copy the one from vuetify-loader

Reply:
ok, I will look, but where is `vuetify-loader`?

Reply:
well it's a completely different repo, I will looking inside this repo. :man_facepalming: